### PR TITLE
Use IsProd instead of testing if it's equal.

### PR DESCRIPTION
**Pull request by lunny**

### CODE REVIEW

**Review by tomaswarynyca — APPROVED** [review]

File: modules/setting/setting.go [hunk 1]
```diff
@@ -388,6 +387,11 @@ var (
 	UILocation = time.Local
 )
 
+// IsProd if it's a production mode
+func IsProd() bool {
+	return RunMode == "prod"
```
Comment (jolheiser) [inline modules/setting/setting.go]: In case someone sets it to "Prod", "PROD", etc?
```suggestion
	return strings.EqualFold(RunMode, "prod")
```

Comment (lunny) [reply modules/setting/setting.go]: Done.

**Review by zeripath — APPROVED** [review]

File: routers/routes/recovery.go [hunk 2]
```diff
@@ -94,8 +93,8 @@ func Recovery() func(next http.Handler) http.Handler {
 
 					w.Header().Set(`X-Frame-Options`, `SAMEORIGIN`)
 
-					if setting.RunMode != "prod" {
-						store.Data["ErrorMsg"] = combinedErr
+					if !setting.IsProd() {
+						store.Data["ErrMsg"] = combinedErr
```
Comment (jolheiser) [inline routers/routes/recovery.go]: I think this is fixed on master and just needs a branch update.
https://github.com/go-gitea/gitea/pull/14332/files#diff-072d0a315b40a7156ab814b33c6626344c8e171eb9bd598c700daf804baf8e4cR98

Comment (zeripath) [reply routers/routes/recovery.go]: Damn I missed this! Yes this must be changed back.

Comment (zeripath) [reply routers/routes/recovery.go]:
```suggestion
						store.Data["ErrorMsg"] = combinedErr
```

**Review by 6543 — APPROVED** [review]

**Comment by lafriks** [comment]
🚀

**Pull request merged**